### PR TITLE
Various EN stringtable fixes/cleaning/formatting

### PR DESCRIPTION
**Pull request by Ouaz**

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -11790,7 +11794,7 @@ Concentration Camps
 BLD_CONC_CAMP_DESC
 '''Decreases the planet's population at a rate of 3 per turn, increases target [[metertype METER_INDUSTRY]] by 5 times the planet's population and sets the target [[metertype METER_HAPPINESS]] to 0.
 
-Ridding a planet of an undesirable species allows the introduction of a more suited species. By creating a planet-wide network of concentration camps designed to work the citizens to death, this goal can be achieved quickly and efficiently. This building can be built on an [[encyclopedia OUTPOSTS_TITLE]].'''
+Ridding a planet of an undesirable species allows the introduction of a more suited species. By creating a planet-wide network of concentration camps designed to work the citizens to death, this goal can be achieved quickly and efficiently. [[BUILDING_AVAILABLE_ON_OUTPOSTS]].'''
```
Comment (MatGB) [inline default/stringtables/en.txt]: This is actually incorrect, not your fault, they have a minimum population requirement of 3.

**Review by MatGB — COMMENTED** [review]
Apart from C Camps, it looks good to me.

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -5363,7 +5369,7 @@ Design Window
...
+Designs and ship parts can be [[PRODUCTION_WND_AVAILABILITY_AVAILABLE]], [[PRODUCTION_WND_AVAILABILITY_UNAVAILABLE]] or [[PRODUCTION_WND_AVAILABILITY_OBSOLETE]].  [[PRODUCTION_WND_AVAILABILITY_AVAILABLE]] designs and parts are unrestricted for use by the empire.  [[PRODUCTION_WND_AVAILABILITY_UNAVAILABLE]] designs and parts are restricted in some way. Some unavailable designs and parts may be unlocked by researching associated [[encyclopedia ENC_TECH]]. [[PRODUCTION_WND_AVAILABILITY_OBSOLETE]] designs are obsoleted/unobsoleted at will by the empire.  [[PRODUCTION_WND_AVAILABILITY_OBSOLETE]] designs will not appear in the [[encyclopedia PRODUCTION_WINDOW_ARTICLE_TITLE]] and only appear in [[DESIGN_WINDOW_ARTICLE_TITLE]] when the "[[PRODUCTION_WND_AVAILABILITY_OBSOLETE]]" filter is toggled to display them.  Ship parts can not be obsoleted.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Why the double spaces after periods, selectively in this block?

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -5372,7 +5378,7 @@ There are four sub windows:  [[DESIGN_WND_STARTS]], [[DESIGN_WND_PART_PALETTE_TI
...
 
-Use the [[DESIGN_WND_SAVED_DESIGNS]] designs right click context menu to copy a single or all saved designs to the [[DESIGN_WND_FINISHED_DESIGNS]] designs. Ctrl click also copies a single design to the [[DESIGN_WND_FINISHED_DESIGNS]] designs.  Permanently remove a saved design with "[[DESIGN_WND_DELETE_SAVED]]".  Check "[[OPTIONS_ADD_SAVED_DESIGNS]]" to automaitcally import all the [[DESIGN_WND_SAVED_DESIGNS]] designs at the start of a new game.
+Use the [[DESIGN_WND_SAVED_DESIGNS]] designs right click context menu to copy a single or all saved designs to the [[DESIGN_WND_FINISHED_DESIGNS]] designs. Ctrl click also copies a single design to the [[DESIGN_WND_FINISHED_DESIGNS]] designs.  Permanently remove a saved design with "[[DESIGN_WND_DELETE_SAVED]]".  Check "[[OPTIONS_ADD_SAVED_DESIGNS]]" to automatically import all the [[DESIGN_WND_SAVED_DESIGNS]] designs at the start of a new game.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: also here... Most other test seemingly doesn't use them. Would be good to standardize (probably on single spaces).

**Comment by Ouaz** [comment]
@geoffthemedio 
> Why the double spaces after periods, selectively in this block?

> Would be good to standardize (probably on single spaces).

Yeah, some (british?) FO contributors use double spaces after a period. I has been debated on another PR but not resolved ... It's an old habit inherited from... typewriters! 
http://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period
https://www.accessc.com/articles/one-space-not-two-spaces-after-period.html

As a french guy, I use obviously the `french spacing` (single space after a period):
https://en.wikipedia.org/wiki/History_of_sentence_spacing#French_and_English_spacing

I also think this shoud be standardized through all the EN stringtable, because there's 67 double-spacings after periods in the file (should require its own PR, IMO)

EDIT: given Mat answers below, I can add a commit which removes the double-spaces. Let me know, folks.

**Comment by MatGB** [comment]
Possibly added by mistake by me at some point, when I was taught to touch type double spacing was the norm, and even though I learned it was pointless more than a decade ago they sometimes creep in when I'm working quickly, there's no need for them at all but that doesn't stop my thumbs.

**Comment by geoffthemedio** [comment]
Removing all the double spaces sounds good. Otherwise probly fine.

**Pull request merged**